components/browser/state/src/main/java/mozilla/components/browser/state/state/ContainerState.kt

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+/**
+ * Value type that represents the state of a container also known as a contextual identity.
+ *
+ * @property contextId The session context ID also known as cookie store ID for the container.
+ * @property name Name of the container.
+ * @property color The color for the container. This can be shown in tabs belonging to this container.
+ * @property icon The icon for the container.
+ */
+data class ContainerState(
+    val contextId: String,
+    val name: String,
+    val color: Color,
+    val icon: Icon
+) {
```

Comment:
Curious if this should be a Parcelable from comparing it to `SitePermissions`?

Reply:
No, I do not think so. I can't see any need for passing instances around inside a bundle (e.g. from activity to activity). Everything can talk to the store directly. If we need to pass anything around then it's probably the id and then the receiver can lookup the right state from the store with that id.

Reply:
Thank you that is very insightful about Parcelable.